### PR TITLE
CommonJS support (Browserify, stitch, etc...)

### PR DESCRIPTION
**Pull request by ericelliott**
Export the module API!

### CODE REVIEW

File: lightningjs-bootstrap.js [hunk 1]
```diff
@@ -218,4 +218,7 @@ window.lightningjs || (function(window, parentLightningjs){
         })
     }
 
-})(window, window.parent.lightningjs);
+    if (module && module.exports) {
```
Comment (mjpizz) [inline lightningjs-bootstrap.js]: I think you meant `if (typeof module !== "undefined" && module.exports)` (avoids ReferenceError in non-CommonJS envs when `module` is not defined)

**Comment by mjpizz** [comment]
Thanks Eric!  Good call.  I assume you're using this like:

``` javascript
var lightningjs = require("lightningjs");
lightningjs.provide({foo: ...})
```

or something along those lines?

File: lightningjs-bootstrap.js (hunk 1 above)
Comment (ericelliott) [inline lightningjs-bootstrap.js]: Oops. Yes, of course.  =)

**Comment by ericelliott** [comment]
```
var lightningjs = require("lightningjs");
lightningjs.provide({foo: ...});
```

That's pretty much exactly what we're doing. =)

**Comment by mjpizz** [comment]
sweet, looks good with the fix.

**Pull request merged**